x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -1154,6 +1173,34 @@ LogEntryBuilder withRequestBody(DeletePrivilegesRequest deletePrivilegesRequest)
             return this;
         }
 
+        LogEntryBuilder withRequestBody(CreateServiceAccountTokenRequest createServiceAccountTokenRequest) throws IOException {
+            logEntry.with(EVENT_ACTION_FIELD_NAME, "create_service_account_token");
+            XContentBuilder builder = JsonXContent.contentBuilder().humanReadable(true);
+            builder.startObject()
+                .startObject("service_account_token")
```

Comment:
I noticed that when the token is user it is audited as `service_token` but when it is created or deleted it is `service_account_token`. I don't have any preference but I think it's best to pick one.

Reply:
Thanks for catching it. I meant to use `service_token` for all places that are user visible. But sometimes the word `service_account_token` almost types itself :)